packages/terra-switch/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "terra-switch",
+  "main": "lib/Switch.js",
+  "version": "0.0.0",
+  "description": "The terra-switch component provides users a way to toggle between state ON and OFF.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "switch",
+    "Cerner",
+    "Terra",
+    "terra-switch",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "scripts": {
+    "compile": "babel --root-mode upward src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "precompile": "rm -rf lib",
+    "test": "npm run jest && npm run wdio",
```

Comment:
"test" should also include the linter check "npm run lint"

Reply:
Added : 48fce2d9ca73773ac6f9891feaa7f2a56b36f864